src/main/java/edu/greenblitz/robotName/subsystems/swerve/Modules/kazaSwerveModule/KazaSwerveModule.java

```diff
@@ -127,8 +113,8 @@ public void updateInputs(SwerveModuleInputsAutoLogged inputs) {
         inputs.linearVoltage = linearMotor.getAppliedOutput();
         inputs.angularVoltage = angleMotor.getAppliedOutput();
 
-        inputs.linearVelocity = Conversions.convertRPMToRadsPerSec(linearMotor.getEncoder().getVelocity() * KazaSwerveConstants.angleTicksToWheelToRPM);
-        inputs.angularVelocity = angleMotor.getEncoder().getVelocity() * KazaSwerveConstants.angleTicksToRadians;
+        inputs.linearVelocity = Conversions.convertRPMToRadsPerSec(linearMotor.getEncoder().getVelocity() * KazaSwerveConstants.ANGULAR_TICKS_TO_WHEEL_TO_RPM);
+        inputs.angularVelocity = angleMotor.getEncoder().getVelocity();
 
         inputs.linearMetersPassed = linearMotor.getEncoder().getPosition();
         inputs.angularPositionInRads = angleMotor.getEncoder().getPosition();
```

Comment:
Out of scope for the CR but

![image](https://github.com/GreenBlitz/Robot-Template/assets/80216549/e246e90b-3538-4522-813d-821d51853910)

Rads?